src/shell/views/Shell/Shell.tsx

```diff
@@ -118,7 +131,6 @@ export default memo(function Shell() {
           <Sentry.ErrorBoundary
             fallback={() => <AppError />}
             beforeCapture={(scope) => {
-              scope.setLevel(Severity.Fatal);
```

Comment:
This is used to determine a sentry error that actually crashed the app. Any reason this was removed?